web/src/main/java/com/bazaarvoice/emodb/web/resources/blob/BlobStoreResource1.java

```diff
@@ -143,7 +143,7 @@ public Meter load(String key) throws Exception {
                                 .addTag("apiKey", key)
                                 .build()
                                 .encode();
-                        return _metricRegistry.meter(MetricRegistry.name(metric));
+                        return _metricRegistry.meter(MetricRegistry.name("bv.emodb.blob.BlobStoreResource1", metric));
```

Comment:
question - should the concatenation of the metric name go to line 142 :?

Reply:
result would be the same, but will move